W-19120572-console-logging (#389)
mulesoft/docs-hosting: cloudhub-2/modules/ROOT/pages/ch2-integrate-log-system.adoc

* File appenders, such as FileAppender, RollingFileAppender, AnypointMonitoringFileAppender, and RandomAccessFileAppender, are removed automatically.
* When using custom Log4j configurations, both Mule runtime engine and application logs remain available in Runtime Manager.
* You can forward tracing module logs using only third-party appenders. You cannot update the logging patterns to send them to Anypoint Monitoring.

> * Console logging is disabled by default in CloudHub 2.0. If you change log levels or appenders in a custom `log4j2.xml` and don't see the changes in the logs, ensure that your configuration uses an appender that CloudHub 2.0 collects; for example, a third-party appender that streams to your logging system. Log output from a Console appender alone doesn't appear in Runtime Manager unless  you explicitly enable console logging for your deployment.

Author: ```suggestion
* Console logging is disabled by default in CloudHub 2.0. If you change log levels or appenders in a custom `log4j2.xml` and don't see the changes in the logs, use an appender that CloudHub 2.0 collects, such as a third-party appender that streams to your logging system. Log output from a Console appender alone doesn't appear in Runtime Manager.
```

> * Console logging is disabled by default in CloudHub 2.0. If you change log levels or appenders in a custom `log4j2.xml` and don't see the changes in the logs, ensure that your configuration uses an appender that CloudHub 2.0 collects (for example, a third-party appender that streams to your logging system). Log output from a Console appender alone does not appear in Runtime Manager unless console logging is explicitly enabled for your deployment.

Author: ```suggestion
* Console logging is disabled by default in CloudHub 2.0. If you change log levels or appenders in a custom `log4j2.xml` and don't see the changes in the logs, ensure that your configuration uses an appender that CloudHub 2.0 collects; for example, a third-party appender that streams to your logging system. Log output from a Console appender alone doesn't appear in Runtime Manager unless  you explicitly enable console logging for your deployment.
```

== Example: Create a Log4j Configuration Using Splunk

[IMPORTANT]
====

> Console logging is disabled by default in CloudHub 2.0. Changes to log levels or appenders in your custom log4j2.xml might not appear in the logs in Runtime Manager if you rely only on a Console appender. To see your custom Log4j configuration take effect, use an appender that streams logs to an external system, such as Splunk, or enable log integration for your deployment so that console output is collected.

Author: ```suggestion
Console logging is disabled by default in CloudHub 2.0. If you rely only on a Console appender, changes to log levels or appenders in your custom log4j2.xml might not appear in the logs in Runtime Manager. To apply your custom Log4j configuration, use an appender that streams logs to an external system, such as Splunk, or enable log integration for your deployment to collect console output.
```

> Console logging is disabled by default in CloudHub 2.0. Changes to log levels or appenders in your custom `log4j2.xml` might not appear in the logs in Runtime Manager if you rely only on a Console appender. To see your custom Log4j configuration take effect, use an appender that streams logs to an external system (such as Splunk) or ensure that log integration is enabled for your deployment so that console output is collected.

Author: ```suggestion
Console logging is disabled by default in CloudHub 2.0. Changes to log levels or appenders in your custom log4j2.xml might not appear in the logs in Runtime Manager if you rely only on a Console appender. To see your custom Log4j configuration take effect, use an appender that streams logs to an external system, such as Splunk, or enable log integration for your deployment so that console output is collected.
```
